src/main/java/org/wso2/carbon/module/swiftiso20022/constants/ConnectorConstants.java

```diff
@@ -74,13 +82,22 @@ private ConnectorConstants() {
     public static final String BLOCK03_VALIDATION_FLAG = "Validation Flag";
     public static final String BLOCK03_END_TO_END_REFERENCE = "End to End Reference";
     public static final String BLOCK03_SERVICE_TYPE_IDENTIFIER = "Service Type Identifier";
+    public static final String BLOCK03_MESSAGE_INPUT_REFERENCE = "Message Input Reference";
```

Comment:
The same constant is defined in L71. We can define these as common constants. No need of having the block no for the constant name.

Reply:
Changed in commit 4daedbd4cd0fb94c36373fe83a8c13d9123a4efb